src/main/kotlin/com/zopa/ktor/opentracing/ThreadContextElementScopeManager.kt

```diff
@@ -34,7 +35,9 @@ internal class CoroutineThreadLocalScope: Scope {
             return
         }
 
-        spanStack.pop()
+        if (spanStack.isNotEmpty()) {
+            spanStack.pop()
+        }
```

Comment:
Eliminates the possibility of having a `EmptyStackException` for whatever reason